src/services/userService.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { User } = require('../models/User.model');
+
+const userService = {
+  async createUser({ name }) {
+    const newUser = await User.create({ name });
+
+    return newUser;
+  },
+
+  async getAllUsers(filters = {}) {
```

Comment:
Same here: `getAllUsers(where = {})`, and remove next `where` declaration, as it's not necessary